s3torchconnector/src/s3torchconnector/s3iterable_dataset.py

```diff
@@ -83,6 +87,7 @@ def from_prefix(
         Raises:
             S3Exception: An error occurred accessing S3.
         """
+        log.info(f"Building {cls.__name__} from_prefix")
```

Comment:
Should we also include the prefix here? 
For objects_uris is too much to log, but prefix might be useful.